crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -698,46 +766,15 @@ where
         Ok(())
     }
 
+    // TODO:
```

Comment:
This can be turned into a note when the unnecessary code/refactor is deleted/reverted.